app/src/lib/git/rev-parse.ts

```diff
@@ -52,6 +52,32 @@ export async function getTopLevelWorkingDirectory(
   return Path.resolve(path, relativePath)
 }
 
+export async function checkIfRepositoryIsBare(
```

Comment:
:art: can we rename this function to `isBareRepository` - this feels more On Brand™ alongside `isGitRepository` which is also defined in this file.